package.json

```diff
@@ -27,44 +27,49 @@
     "components"
   ],
   "files": [
-    "preact-components"
+    "src"
   ],
   "jspm": {
     "registry": "npm",
     "jspmPackage": true,
+    "format": "esm",
     "directories": {
-      "preact-components": "preact-components"
+      "lib": "src"
+    },
+    "peerDependencies": {
+      "preact": "7.1.0",
+      "preact-compat": "3.13.1"
     }
   },
   "author": "Fabian Weber",
   "license": "MIT",
   "devDependencies": {
     "babel-loader": "6.2.10",
-    "babel-plugin-transform-react-jsx": "6.22.0",
+    "babel-plugin-transform-class-properties": "^6.23.0",
+    "babel-plugin-transform-react-jsx": "^6.23.0",
     "babel-preset-es2015": "6.22.0",
-    "babel-preset-stage-1": "^6.5.0",
     "chai": "3.5.0",
     "css-loader": "0.26.1",
-    "eslint": "3.15.0",
-    "eslint-config-airbnb": "14.1.0",
-    "eslint-plugin-import": "2.2.0",
-    "eslint-plugin-jsx-a11y": "4.0.0",
-    "eslint-plugin-react": "6.10.0",
+    "eslint": "^3.15.0",
+    "eslint-config-airbnb": "^14.1.0",
+    "eslint-plugin-import": "^2.2.0",
+    "eslint-plugin-jsx-a11y": "^4.0.0",
+    "eslint-plugin-react": "^6.10.0",
     "lite-server": "^2.2.0",
     "mocha": "3.2.0",
     "mocha-webpack": "0.7.0",
     "node-sass": "4.5.0",
     "pre-commit": "1.2.2",
+    "preact": "7.1.0",
+    "preact-compat": "3.13.1",
     "sass-loader": "5.0.1",
     "style-loader": "0.13.1",
     "webpack": "2.2.1",
     "webpack-dev-server": "2.3.0",
-    "wholesale-styleguide-scss": "git+ssh://git@github.bus.zalan.do/WholesaleDesignSystem/Styleguide-SCSS.git"
+    "wholesale-styleguide-scss": "git+ssh://git@github.bus.zalan.do/WholesaleDesignSystem/Styleguide-SCSS.git#dropdown-adjustments"
   },
-  "dependencies": {
+  "peerDependencies": {
     "preact": "7.1.0",
-    "preact-compat": "3.13.1",
-    "react": "15.4.2",
```

Comment:
But then they are needed only as devDependencies but not as dependencies or peerDependencies. I gonna add them